website/content/docs/security/security-models/core.mdx

```diff
@@ -275,16 +275,22 @@ environment and adapt these configurations accordingly.
 - **Secure UI Access** - Access to Consul’s builtin UI can be secured in various ways:
 
   - **mTLS** - Enabling the HTTPS with mutual TLS authentication is recommended, but requires extra tooling to terminate
-    the mTLS connection, preferably on an operator's local machine using a proxy script.
+    the mTLS connection, preferably on an operator's local machine using a proxy script. To do this, follow the tutorial on
+    [Configuring the Consul UI for HTTPS](https://learn.hashicorp.com/tutorials/consul/tls-encryption-openssl-secure#configure-the-consul-ui-for-https)
+    and in Step 2 use "Option 1: add a client certificate to your browser".
 
   - **TLS** - Enabling the HTTPS is recommended where mTLS may not be required for UI access, such as when ACLs are
-    configured with a default deny.
+    configured with a default deny. To do this, follow the tutorial on
+    [Configuring the Consul UI for HTTPS](https://learn.hashicorp.com/tutorials/consul/tls-encryption-openssl-secure#configure-the-consul-ui-for-https)
+    and in Step 2 use "Option 2: verify_incoming_rpc".
 
   - **ACL** - ACLs with a default deny policy enables safer UI access by preventing unauthorized access to sensitive
-    components within the cluster.
+    components within the cluster. To do this, follow the tutorial on
+    [Securing Consul with Access Control Lists (ACLs)](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production),
```

Comment:
Hey 👋 , wondered if now might be a good chance to mention that some potentially sensitive components cannot be protected via the ACL system. I'm not entirely sure how sensitive things like dc names are for example (I would guess some folks would class them as sensitive) and there might be the other odd thing also.

Reply:
Meaning: some information is always visible on the UI even if someone lacks a token that provides any access?

Are we able to list what those potentially sensitive things are? I would want to be specific (so we answer more questions than we raise).

Reply:
@jkirschner-hashicorp You can play with a Consul cluster in this tutorial to explore what is exposed. Just click on the "Show Terminal" button to get started.
https://learn.hashicorp.com/tutorials/consul/access-control-setup-production?in=consul/security

Reply:
Yeah so DC names at least are always visible via the API even if you don't have a token, I only know this as thats an endpoint the UI uses, not sure if there are others or not.

Reply:
@johncowen : I like that idea, however I want to address it outside the scope of this PR. I spent time today tracing through the HTTP API endpoint docs to note which endpoints cannot be protected by ACLs and what information they provide. I think it's worth putting up on the docs site and cross-referencing from here, but I don't want to couple it to this PR / delay it further.